backend/fms_core/template_importer/row_handlers/transfer/transfer.py

```diff
@@ -17,7 +18,7 @@ def process_row_inner(self, source_sample_info, resulting_sample_info, process_m
 
         if original_sample:
             if source_sample_info['depleted']:
-                original_sample.depleted = True if source_sample_info['depleted'] == "YES" else False
+                original_sample.depleted = original_sample.depleted or check_truth_like(source_sample_info['depleted'])
```

Comment:
This reminds me that we should add a way of checking whether the sample is depleted or not before attempting a transfer or extraction. Also, there is the case where this cell is not filled in the spreadsheet; it should have no impact (no change) to the original_sample; we should be careful not to have it set to False (not depleted) if the field is None (not filled)